cmd/config.go

```diff
@@ -114,7 +116,12 @@ func Config(c *git.Client, args []string) {
 		defer outfile.Close()
 		config.WriteFile(outfile)
 		os.Exit(0)
-		return
+        case "list":
+                list := config.GetConfigList()
+                for _, entry := range list {
+                        fmt.Printf("%s\n", entry)
+                }
+                os.Exit(0)
```

Comment:
I think this should just return (and the panic's be converted to return an error), this was one of the earlier commands implemented before the package conventions were as well defined.